kubernetes/ansible/roles/secor-deploy/tasks/main.yaml

```diff
@@ -0,0 +1,17 @@
+- name: template values.yaml file
+  vars:
+    secor_job_list: "{{ job_names_to_deploy.split(',')|list }}"
```

Comment:
From where are we getting `job_names_to_deploy`? If its jenkins, then jenkins file and the ansible playbook also need to be added in this PR which can accept this as an extra vars.

Reply:
Yes, this is from Jenkins. Jenkinsfile has been added as part of another PR.